### DISCUSSION TITLE
急急急，我的物资丢失在零号空洞内，委托费300菲林！

### DISCUSSION THREAD

**Discussion by FaranCheng** [share121/inter-knot#2647] [其他]
![image](https://github.com/user-attachments/assets/f056db64-38d0-4e59-9559-7dbffa9f90a0)

如题，我的物资丢失了，希望一位技术高超的绳匠帮我完成此次委托！

**Comment by Orangered0801** [comment]
如果给我新增代理人简的话，我会好好考虑的<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >

**Comment by AAAzhaoxiangji** [comment]
这个委托我接了！

**Comment by FaranCheng** [comment]
就你了！

**Comment by Fate114-514** [comment]
我去，脚本狗（悲）

**Comment by Maple013** [comment]
实不相瞒，虽然您看上去非常着急，但300丁尼的委托费用实在是有些……呃，夺少？
# 夺少？？？
